src/map-reduce.lisp

```diff
@@ -16,8 +16,7 @@ When using :keeptemp t without specificing :out the collection is mr.<collection
 			 (when ,finalize (kv "finalize" (jsdef ,finalize)))
 			 (kv "out"      (if ,out ,out (concatenate 'string "mr." ,collection))) 
 			 (kv "verbose"  ,verbose)
-			 (kv "limit"    ,limit)
-			 (kv "keeptemp" ,keeptemp)
+			 (when ,limit (kv "limit" ,limit))
```

Comment:
- "keeptemp" is apparently no longer supported in calls to MongoDB's mapReduce.
- "limit" is apparently no longer allowed to be zero in calls to MongoDB's mapReduce.

Reply:
Well, 0 is "true" in CL, so perhaps `(unless (zerop limit) ...)` or `(when (> limit 0) ...)` ?